### PR TITLE
media-fonts/source-pro: version bump, update to source-sans-pro-2.045R-ro-1.095R-it / source-serif-pro-2.010R-ro-1.010R-it

### PR DESCRIPTION
**Pull request by shimataro**
Update to [source-sans-pro-2.045R-ro-1.095R-it](https://github.com/adobe-fonts/source-sans-pro/releases/tag/2.045R-ro%2F1.095R-it) / [source-serif-pro-2.010R-ro-1.010R-it](https://github.com/adobe-fonts/source-serif-pro/releases/tag/2.010R-ro%2F1.010R-it)

### CODE REVIEW

File: media-fonts/source-pro/source-pro-20191209.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (ajakk) [inline media-fonts/source-pro/source-pro-20191209.ebuild:4]: Can we update to EAPI=7 at this point?

File: media-fonts/source-pro/source-pro-20191209.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+inherit font
```
Comment (ajakk) [inline media-fonts/source-pro/source-pro-20191209.ebuild:5]: Newline in between EAPI and inherit lines, please.

**Review by ajakk — CHANGES_REQUESTED** [review]
Ping @gentoo/fonts

**Comment by ajakk** [comment]
I guess no one is there.

**Comment by shimataro** [comment]
Very sorry for late reply...
Since both fonts have new releases, I will modify ebuild file and create another PR later.

<https://github.com/adobe-fonts/source-serif/releases>
<https://github.com/adobe-fonts/source-sans/releases>

**Pull request closed without merging**